script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -16,22 +16,22 @@ My broken fingers have healed, the bleeding has stopped, and my wounds have clos
 [sha:2827c0a158e16f321fcfc4b519c869515f16b2b1]{
 -- Page 2, Offset 33069.
 -- 人間離れしすぎていて、自分のものとは思えない。
-It's so far detached from a normal human's that I can hardly believe this body is mine.
+It's so unlike an ordinary human hand that I can't even think of it as my own.
```

Comment:
I feel like keeping the 'not human' is important. Otherworldly is kinda vague in what it can entail, and honestly reads with a positive spin half the time.
> > It's such a viscerally inhuman hand that I can't ... 
> This grisly evidence of its inhuman nature makes it impossible for me to think of it as my own.

Reply:
It's specifically the way his hand has restored itself which is the 人間離れ part, so there's no need to bring in the hand again. I do think the point about human being there is valid, but it should just be something like

>It's so inhuman that...

because he's just referring back to the previous line.